src/components/EthExchanges.tsx

```diff
@@ -543,9 +558,15 @@ const EthExchanges = () => {
   const exchangesArray = Object.keys(exchanges)
   const walletProvidersArray = Object.keys(walletProviders)
   // Construct arrays for CardList
-  let filteredExchanges = []
-  let filteredWalletProviders = []
-  let filteredWallets = []
+  let filteredExchanges: Array<{
+    title: string
+    description: string
+    link: string
+    image: string
+    alt: string
+  }> = []
```

Comment:
Looks like this type is the same we are using for the `Wallet` interface. Maybe we can rename it and use it in both places.